### PR TITLE
feat: add dart package

### PR DESCRIPTION
**Pull request by akshdeep-singh**
### Summary
Add dart package for Sant Lipi.

### Test
-

### CODE REVIEW

**Comment by bhajneet** [comment]
Awesome work! Thank you for this. Just confirming you've tested the variable font in the example? And that variable fonts are well supported in the different environments that this package can be used in (e.g. dart/flutter and android/web/desktop)?

My one suggestion (if variable fonts are well supported) is to drop the individual weights to reduce the package size. Even if variable fonts are not well supported, I would rather see a separate package for the individuals, to cut down on package size for those that can use VF properly.

**Comment by bhajneet** [comment]
In addition, please remove the lib folder, we can do that in CI/CD

**Comment by akshdeep-singh** [comment]
Do you have any package name suggestions?

sant_lipi_vf, sant_lipi?

**Comment by bhajneet** [comment]
If VF is strongly supported then only the `sant_lipi` package. Is there any need for a second package? I'm guessing it's use-case is limited and on those rare ocassions people can just manually use the interpolated fonts. That's how I've set up the npm/javascript package for now.

**Comment by akshdeep-singh** [comment]
VF is supported by Flutter.
https://docs.flutter.dev/ui/design/text/typography#variable-fonts

However, it seems the support has been added in recent years, and old Flutter sdks may not be able to use it.
https://github.com/flutter/flutter/issues/33709

I have used variable fonts on Flutter on Android and it works fine.

I have tested this package on Flutter Web, and is working great.

File: packages/pubdev/sant_lipi/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: sant_lipi
+description: A unicode font for extraordinary Gurmukhi.
+version: 1.0.0
+repository: https://github.com/shabados/SantLipi
+topics:
+  - gurmukhi
+  - font
```
Comment (bhajneet) [inline packages/pubdev/sant_lipi/pubspec.yaml:7]:
```suggestion
topics:
  - font
  - gurmukhi
  - punjabi
```

File: packages/pubdev/sant_lipi/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Sant Lipi
+
+A [unicode](<https://en.wikipedia.org/wiki/Gurmukhi_(Unicode_block)>) font for extraordinary [Gurmukhi](https://en.wikipedia.org/wiki/Gurmukhi).
+
+Sant Lipi is a [variable font](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide) with the purpose of fully expressing the unique characters and combos found in the [database](https://github.com/shabados/database) at [Shabad OS](https://shabados.com/).
+
+Please see Sant Lipi's [GitHub](https://github.com/shabados/SantLipi) repo to learn more.
+
+## Installation
```
Comment (bhajneet) [inline packages/pubdev/sant_lipi/README.md:9]:
```suggestion
## Installation

Import the font
```

File: packages/pubdev/sant_lipi/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+```shell
+dart pub get sant_lipi
+```
+
+## Usage
+
+Add the font in `pubspec.yaml` of your project.
```
Comment (bhajneet) [inline packages/pubdev/sant_lipi/README.md:17]:
```suggestion
Declare the font in the pubspec
```

File: packages/pubdev/sant_lipi/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```yaml
+flutter:
+  fonts:
+    - family: SantLipi
+      fonts:
+        - asset: packages/sant_lipi/SantLipi.ttf
```
Comment (bhajneet) [inline packages/pubdev/sant_lipi/README.md:23]:
```suggestion
        - asset: packages/sant_lipi/SantLipi-VF.ttf
```

File: packages/pubdev/sant_lipi/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+Sant Lipi supports variable font weights in the range `100-900`.
```
Comment (bhajneet) [inline packages/pubdev/sant_lipi/README.md:26]:
```suggestion
## Usage

Note: The font weights available are from 100 to 900 (e.g. the `400` below could be `581.321`).
```

File: packages/pubdev/sant_lipi/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+## Installation
+
+Import the font
+
+```shell
+dart pub get sant_lipi
+```
+
+Declare the font in the pubspec
```
Comment (bhajneet) [inline packages/pubdev/sant_lipi/README.md:17]:
```suggestion
Declare the font in the pubspec

```

**Pull request merged**